go.mod

```diff
@@ -297,13 +297,6 @@ replace (
 	// dragonberry
 	github.com/confio/ics23/go => github.com/confio/ics23/go v0.9.0
 
-	// enforce same SDK and IBC on all dependencies
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.13-ics
-	github.com/cosmos/ibc-go/v4 => github.com/cosmos/ibc-go/v4 v4.2.0
```

Comment:
Is this IBC version reflected above?

Reply:
so not v4.3.0 ? please see this pr https://github.com/cosmos/gaia/pull/2185/files

Reply:
This is a different pr just to tidy everything, we can combine